x-pack/plugins/canvas/canvas_plugin_src/functions/browser/location.js

```diff
@@ -12,8 +14,10 @@ export const location = () => ({
   context: {
     types: ['null'],
   },
-  help:
-    "Use the browser's location functionality to get your current location. Usually quite slow, but fairly accurate",
+  help: i18n.translate('xpack.canvas.functions.locationHelpText', {
```

Comment:
`Text` is redundant, considering this a translated string.
```suggestion
  help: i18n.translate('xpack.canvas.functions.locationHelp', {
```